File: python/class 3/question 1.py
print("Welcome to the power calculator!")
base=float(input("Enter the base: "))
exponent=float(input("Enter the exponent: "))
print(f"{base} to the power of {exponent} is {base**exponent}")








#alternatively you could do 
print(f"Welcome to the power calculator!\n{(base:=float(input('Enter the base: ')))} to the power of {(exponent:=float(input('Enter the exponent: ')))} is {base**exponent}")

File: java/class 3/Question1.java
import java.util.Scanner;

public class Question1{
  public static void main(String[] args){
    Scanner input = new Scanner(System.in);
    System.out.println("Welcome to the power calculator!");
    System.out.print("Enter a number: ");
    double base = input.nextDouble();
    System.out.print("Enter another number: ");
    double exponent = input.nextDouble();
    input.close();
    System.out.println(base + " to the power of " + exponent + " is " + Math.pow(base,exponent));
  }
}
